tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -524,20 +533,222 @@ def test_01_multiattach_volumes(self):
+            except Exception as e:
+                LOG.error(f"Exception: {e}")
+                raise Exception("Create workload with volume booted vm")
+            if (self.wid is not None):
+                self.wait_for_workload_tobe_available(self.wid)
+                self.workload_status = self.getWorkloadStatus(self.wid)
+                if (self.workload_status == "available"):
+                    reporting.add_test_step("Create workload with volume booted vm", tvaultconf.PASS)
+                    tests[0][1] = 1
+                    reporting.test_case_to_write()
+                else:
+                    raise Exception("Create workload with volume booted vm")
+            else:
+                raise Exception("Create workload with volume booted vm")
+
+            reporting.add_test_script(tests[1][0])
+            self.snapshot_id = self.workload_snapshot(self.wid, True)
+            self.wait_for_workload_tobe_available(self.wid)
+            self.snapshot_status = self.getSnapshotStatus(self.wid,
+                                                          self.snapshot_id)
+            if (self.snapshot_status == "available"):
+                reporting.add_test_step("Create full snapshot", tvaultconf.PASS)
+                self.mount_path = self.get_mountpoint_path(tvaultconf.tvault_ip[0], tvaultconf.tvault_username,
+                                                           tvaultconf.tvault_password)
+                self.snapshot_found = self.check_snapshot_exist_on_backend(
+                    tvaultconf.tvault_ip[0], tvaultconf.tvault_username,
+                    tvaultconf.tvault_password, self.mount_path,
+                    self.wid, self.snapshot_id)
+                LOG.debug(f"snapshot_found: {self.snapshot_found}")
+                if self.snapshot_found:
+                    reporting.add_test_step("Verify snapshot existence on target backend", tvaultconf.PASS)
+
+                    tests[1][1] = 1
+                    reporting.test_case_to_write()
+                else:
+                    raise Exception("Verify snapshot existence on target backend")
+            else:
+                raise Exception("Create full snapshot")
+
+            reporting.add_test_script(tests[2][0])
+            md5sums_before_incr = self._add_data_on_instance_and_volume([fip[0]], False)
+            LOG.debug(
+                f"md5sums_before_incr: {md5sums_before_incr}")
+
+            self.snapshot_id2 = self.workload_snapshot(self.wid, False)
+            self.wait_for_workload_tobe_available(self.wid)
+            self.snapshot_status = self.getSnapshotStatus(self.wid,
+                                                          self.snapshot_id2)
+            if (self.snapshot_status == "available"):
+                reporting.add_test_step("Create incremental snapshot", tvaultconf.PASS)
+                self.mount_path = self.get_mountpoint_path(tvaultconf.tvault_ip[0], tvaultconf.tvault_username,
```

Comment:
mount path can be retrieved only once and used throughout the test, instead of multiple calls to fetch same value.

Reply:
removed redundant call